components/enterprise-contract/rekor-local/rekor-namespace.yaml

```diff
@@ -0,0 +1,8 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: rekor
+  annotations:
+    openshift.io/sa.scc.uid-range: 65530-65535
```

Comment:
Let's document why these annotations are needed. 

Also, is this a supported/documented approach, or is this something that is going to bite us later?

Reply:
The only way I can make it work with Rekor helm charts, by default each namespace gets a range of UIDs, no idea what they are or how to pass to the helm chart.